CHANGELOG.md

```diff
@@ -336,7 +336,7 @@
 * 🎸 API call retries and support for error reporting ([#5](https://github.com/kolplattformen/skolplattformen-app/issues/5)) ([9ed5df2](https://github.com/kolplattformen/skolplattformen-app/commit/9ed5df2e45e8d368d3d4df1c992a0901a512ff6a))
 * 🎸 Build, tag and release ([b71adc5](https://github.com/kolplattformen/skolplattformen-app/commit/b71adc57fbdd4ff4d6fe1b87e7afac90a9d6628c))
 * 🎸 cache busting ([5ce4ddd](https://github.com/kolplattformen/skolplattformen-app/commit/5ce4ddd9d04501a15ce37c9f3378e36b3f3dabfe))
-* 🎸 Cache prefixas med personnummer ([#8](https://github.com/kolplattformen/skolplattformen-app/issues/8)) ([fc146ea](https://github.com/kolplattformen/skolplattformen-app/commit/fc146ea7fc9f705b88bbc75a6cc01b6f2235bfaa))
+* 🎸 Cache prefixes med personnummer ([#8](https://github.com/kolplattformen/skolplattformen-app/issues/8)) ([fc146ea](https://github.com/kolplattformen/skolplattformen-app/commit/fc146ea7fc9f705b88bbc75a6cc01b6f2235bfaa))
```

Comment:
I know that `med personnummer` is Swedish, but Google Translate for Swedish suggests that `prefixas` isn't right.

Reply:
Super!